sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/mongodb/MongoDbReadWriteIT.java

```diff
@@ -187,7 +199,68 @@ public void testWriteAndRead() {
     PCollection<Row> output =
         BeamSqlRelUtils.toPCollection(readPipeline, sqlEnv.parseQuery("select * from TEST"));
 
-    assertEquals(output.getSchema(), SOURCE_SCHEMA);
+    assertEquals(SOURCE_SCHEMA, output.getSchema());
```

Comment:
nit: I generally prefer syntax like `assertThat(output.getSchema(), equalTo(SOURCE_SCHEMA))` since it makes it clear which is the value being tested and which is the expectation.

Reply:
I agree, `assertEquals` is not very intuitive in terms of which parameter is the expected value and which is the actual. Will change in a next commit.